### ISSUE TITLE
[@html-eslint/element-newline] skip option does not work with nested tags

### ISSUE THREAD

**Issue by yeonjuan** [yeonjuan/html-eslint#160]
### html
```html
<!DOCTYPE html>
<html>
    <head>
    </head>
    <body>
     	 <div><li> foo </li><div></div>
        </div>
    </body>
</html>
```
### .eslintrc.json
```json
{
  "rules": {
    "@html-eslint/element-newline": ["error", {"skip":["div"]}]
  }
}
```

### Expected Result

No lint error

### Actual Result

Lint error, the rule's skip options does not work when tags in skip are nested.

> 6:6 - There should be a linebreak after
